### PR TITLE
Bug 1192751 - Fix failure in test_sms.py

### PR DESCRIPTION
**Pull request by mwargers**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=509a8b3215b626522b267ae981d70afc36f53d8e

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=aaddbf1e8b68b9d06c07533337550cd51b3e3f4b

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=aed4089fd178e8a97c8e5a4f4166a56a337aa33f

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b61c17cfd460cf98191565e9698e0e22589143b3

File: tests/python/gaia-ui-tests/gaiatest/apps/messages/regions/new_message.py [hunk 1]
```diff
@@ -29,8 +29,9 @@ def __init__(self, marionette):
         Base.__init__(self, marionette)
         Wait(self.marionette).until(lambda m: self.apps.displayed_app.name == self.name)
         self.apps.switch_to_displayed_app()
-        section = self.marionette.find_element(*self._thread_messages_locator)
-        Wait(self.marionette).until(lambda m: section.location['x'] == 0)
+        element = Wait(self.marionette).until(
+            expected.element_present(*self._panel_conversationview_locator))
+        Wait(self.marionette).until(lambda m: element.rect['x'] == 0 and element.is_displayed())
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/apps/messages/regions/new_message.py]: What's the advantage to use `rect` instead of `location`, here?

Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/apps/messages/regions/new_message.py]: Location is deprecated, see: http://mxr.mozilla.org/mozilla-central/source/testing/marionette/driver/marionette_driver/marionette.py#140

See https://bugzilla.mozilla.org/show_bug.cgi?id=1155746 for replacing it with rect (internally, location is already using rect.

**Pull request merged**